docs/conf.py

```diff
@@ -74,37 +74,37 @@
 
 # There are two options for replacing |today|: either, you set today to some
 # non-false value, then it is used:
-#today = ''
+# today = ''
 # Else, today_fmt is used as the format for a strftime call.
-#today_fmt = '%B %d, %Y'
+# today_fmt = '%B %d, %Y'
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = ["_build"]
 
 # The reST default role (used for this markup: `text`) to use for all
 # documents.
-#default_role = None
+# default_role = None
 
 # If true, '()' will be appended to :func: etc. cross-reference text.
-#add_function_parentheses = True
+# add_function_parentheses = True
 
 # If true, the current module name will be prepended to all description
 # unit titles (such as .. function::).
-#add_module_names = True
+# add_module_names = True
 
 # If true, sectionauthor and moduleauthor directives will be shown in the
 # output. They are ignored by default.
-#show_authors = False
+# show_authors = False
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+pygments_style = "sphinx"
 
 # A list of ignored prefixes for module index sorting.
-#modindex_common_prefix = []
+# modindex_common_prefix = []
 
 # If true, keep warnings as "system message" paragraphs in the built documents.
-#keep_warnings = False
+# keep_warnings = False
```

Comment:
Do we need to make all these changes, if this is just how `sphinx-quickstart` generates them?

Reply:
(there's a difference between "a comment" and "a commented out line", black!)

Reply:
I like this space better (for commented code), but generally I mark docs/conf.py as not part of Black, and didn't do that here. I can restore it and add it to the ignore list if you'd like.

Reply:
I hadn't excluded it properly, fixed.

Reply:
As you prefer. I thought it was weird that this file, generated by Sphinx itself got changed.
But if you want to still have black run on this, maybe a separate PR would be cleaner?

Reply:
No, I usually keep black off of generated files. There was an attempt to do so in the original code (check the history), but I got it wrong. 😳